config/database.yml

```
@@ -60,5 +60,6 @@ production:
     password: <%= (ENV['REPLICA_DB_PASS'] || ENV['DB_PASS'] || '').to_json %>
     host: <%= ENV['REPLICA_DB_HOST'] ||ENV['DB_HOST'] || 'localhost' %>
     port: <%= ENV['REPLICA_DB_PORT'] ||ENV['DB_PORT'] || 5432 %>
-    prepared_statements: <%= ENV['PREPARED_STATEMENTS'] || 'true' %>
+    prepared_statements: <%= ENV['REPLICA_PREPARED_STATEMENTS'] ||ENV['PREPARED_STATEMENTS'] || 'true' %>
```

Minor style issue, not sure why it isn't caught by anything:
```suggestion
    prepared_statements: <%= ENV['REPLICA_PREPARED_STATEMENTS'] || ENV['PREPARED_STATEMENTS'] || 'true' %>
```

---

Do you want me to add the space for all of the ORs?